search/query/boolean.go

```diff
@@ -33,19 +34,17 @@ type booleanQuery struct {
 // Queries.
 // Result documents that ALSO satisfy any of the should
 // Queries will score higher.
-func NewBooleanQuery(must []Query, should []Query, mustNot []Query) *booleanQuery {
+func NewBooleanQuery(must []Query, should []Query, mustNot []Query) *BooleanQuery {
 	return NewBooleanQueryMinShould(must, should, mustNot, 0.0)
 }
 
 // NewBooleanQueryMinShould is the same as
 // NewBooleanQuery, only it offers control of the
 // minimum number of should queries that must be
 // satisfied.
-func NewBooleanQueryMinShould(must []Query, should []Query, mustNot []Query, minShould float64) *booleanQuery {
+func NewBooleanQueryMinShould(must []Query, should []Query, mustNot []Query, minShould float64) *BooleanQuery {
```

Comment:
This might be a separate, independent issue than the API subpackage move...

But, I was wondering whether NewBooleanQueryMinShould() was really popular?  That is, I was thinking that NewBooleanQuery() + SetMinShould() is equivalent?  And we often try to limit the API porcelain "surface area" when we can?